### ISSUE TITLE
Celestia: RangeError: index out of range:

### ISSUE THREAD

**Issue by alexanderludwig** [subquery/subql#2539]
### Prerequisites

* [x ] Are you running the latest version(s)?
* [ x] Have you searched the [documentation for your issue](https://academy.subquery.network/)?
* [ x] Did you search for an existing issue or pull request?

### Description
When running Celestia standard template receiving this issue:
![image](https://github.com/user-attachments/assets/4203ea82-ee74-4f8a-8d8c-c11946413d98)

### Details

These details can help to reproduce the environment the issue is occurring

![image](https://github.com/user-attachments/assets/183d5739-2f9f-4ce9-9482-6c95b12f14a3)

alex@Alexanders-MBP celestia-starter % subql --version
@subql/cli/5.2.3 darwin-arm64 node-v21.7.3

**Local Environment:** [You can get this information from executing `subql version`.]
**Query Version:** [What is the version of the query service?]
**Indexer Version:** [What is the version of the indexer service?]
**Network Details:**
    chainId: "celestia",
      startBlock: 2143040,
    endpoint: [
      "https://celestia-rpc.publicnode.com:443",
    ],

### Steps to Reproduce

**Example project:** 
Just start the cosmos starter with described params.

**Expected behavior:** [What you expected to happen]
Indexing 

**Actual behavior:** [What actually happened]
error message from above

### Any other information

Is there any other information you would like to add?

**Comment by stwiname** [comment]
Hi @alexanderludwig 

This is trying to decode a transaction using [this function](https://github.com/cosmos/cosmjs/blob/main/packages/proto-signing/src/decode.ts#L12) in a block but its not a [RawTx](https://github.com/confio/cosmjs-types/blob/main/src/cosmos/tx/v1beta1/tx.ts#L32). In order to help with this I would need to know the actual type to find a way to fix this.

**Comment by stwiname** [comment]
We have a similar issue that is being worked on but it doesn't seem to apply to your specific issue. https://github.com/subquery/subql-cosmos/pull/280

**Comment by alexanderludwig** [comment]
thanks for having a look at this @stwiname!! can you try running `docker compose up` for Celestia in cosmos-subql-starter? get the issue there without having done any modifications.

**Comment by stwiname** [comment]
Yes, I can reproduce the issue, we use cosmjs for decoding blocks but Celestia has different content in the transactions data. I've found some relevant issues from their organisation.

https://github.com/celestiaorg/celestia-app/issues/2715
https://github.com/celestiaorg/celestia-core/issues/1136
